--- xdg-utils-1.1.3/scripts/xdg-open.in~	2018-05-10 15:02:31.000000000 +0000
+++ xdg-utils-1.1.3/scripts/xdg-open.in	2019-02-20 04:17:16.889458374 +0000
@@ -436,7 +436,7 @@
 
     # if BROWSER variable is not set, check some well known browsers instead
     if [ x"$BROWSER" = x"" ]; then
-        BROWSER=www-browser:links2:elinks:links:lynx:w3m
+        BROWSER=firefox:www-browser:links2:elinks:links:lynx:w3m
         if has_display; then
             BROWSER=x-www-browser:firefox:iceweasel:seamonkey:mozilla:epiphany:konqueror:chromium:chromium-browser:google-chrome:$BROWSER
         fi
